src/VsixTesting.Installer/VsixTesting.Installer.csproj

```diff
@@ -2,11 +2,19 @@
   <PropertyGroup>
     <TargetFramework>net452</TargetFramework>
     <OutputType>Exe</OutputType>
-    <IsPackable>False</IsPackable>
+    <IsPackable>true</IsPackable>
     <Prefer32Bit>true</Prefer32Bit>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <DocumentationFile>$(OutputPath)$(AssemblyName).xml</DocumentationFile>
```

Comment:
💡 With the new project system, it's sufficient to use this:

```suggestion
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Reply:
Nice 👍